SIPS/sip-15.md

```diff
@@ -112,8 +114,144 @@ The rationale for these liquidation mechanisms are:
 ## Test Cases
 
 <!--Test cases for an implementation are mandatory for SIPs but can be included with the implementation..-->
+Given Alice has issued synths with 800 SNX, with a debt balance of 533.33 sUSD and now has a collateralised ratio of 150% and Bob acts as a liquidator with sUSD,
+
+Given the following preconditions:
+
+- liquidation ratio is 150%
+- liquidation cap is 300%
+- liquidation penalty is 12%
+- and liqudatiion delay is set to _2_ weeks.
+
+***
+
+When
+
+- Bob flags Alice address for liquidation
+
+Then
+
+- ✅ It succeeds and adds an liquidation Entry for Alice as flagged
+- ✅ It sets deadline as block.timestamp + liquidation delay of _2_ weeks.
+- ✅ It emits an event accountFlaggedForLiquidation(account, deadline)
```

Comment:
You can put these events in the interface above.

Reply:
I had the events in the Specifications section described in the functions before